src/Time/TimeSteppers/ImexRungeKutta.hpp

```diff
@@ -0,0 +1,71 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <vector>
+
+#include "Time/TimeSteppers/ImexTimeStepper.hpp"
+#include "Time/TimeSteppers/RungeKutta.hpp"
+#include "Utilities/Gsl.hpp"
+
+/// \cond
+class TimeDelta;
+namespace TimeSteppers {
+template <typename T>
+class ConstUntypedHistory;
+template <typename T>
+class MutableUntypedHistory;
+}  // namespace TimeSteppers
+/// \endcond
+
+namespace TimeSteppers {
+/*!
+ * \ingroup TimeSteppersGroup
+ * Intermediate base class implementing a generic IMEX Runge-Kutta
```

Comment:
I added the NASA reference here, since it explains the notation well.  I'll include Hairer in the tutorial, since that has a bit of discussion of properties of implicit methods and Hairer covers that in depth.